components/registry/org.wso2.carbon.registry.indexing/src/main/java/org/wso2/carbon/registry/indexing/AsyncIndexer.java

```diff
@@ -182,6 +183,8 @@ protected void uploadFiles(List<IndexingTask> tasks) throws RegistryException {
     }
 
     protected static class IndexingTask implements Runnable {
```

Comment:
Hi Imesh and Fernando,
Thanks for pointing out. This is fixed in the same PR
Thanks!